examples/worlds/mecanum_drive.sdf

```diff
@@ -0,0 +1,365 @@
+<?xml version="1.0" ?>
+<!--
+  Ignition Gazebo skid steer differential drive plugin demo
+
+  Try sending commands:
+
+    ign topic -t "/model/vehicle_blue/cmd_vel" -m ignition.msgs.Twist -p "linear: {x: 0.5}, angular: {z: 0.05}"
```

Comment:
It would be interesting to use a command that has `y` linear velocity.

Reply:
added in 15416d0